bin/spectacle/args.js

```diff
@@ -115,5 +117,8 @@ const parse = async argv => {
 };
 
 module.exports = {
-  parse: () => parse(args())
+  parse: async () => {
+    const parsedArgs = await init.promptArgs? await init.promptArgs : args();
```

Comment:
We're going to refactor this per the rest of the comments, but `await init.promptArgs` as it is currently coded always returns truthy, so the conditional isn't actually used here as it is equivalent to:

```js
const parsedArgs = { one: 1, two: 2 } ? { one: 1, two: 2 } : args();
```